frontend/src/components/SearchBar/FindMorePropertiesForm.tsx

```diff
@@ -102,7 +121,13 @@ const FindMorePropertiesForm = <T extends any>(props: any) => {
         </a>
       </p>
       <FormSection>
-        <StyledRow style={{ marginLeft: 140, paddingTop: 10 }}>
+        {displayError && (
+          <InvalidFeedback>
```

Comment:
Do you need the displayError && part? or is it enough that the DisplayError component won't display unless the touched and error values within formik are present?

Reply:
It seems so because the user may not be interacting with those checkboxes so they will never be touched. And with the errorPrompt prop enabled it will display even without submit so I need a little bit of extra logic

Reply:
(if they try to submit without choosing an option)